anoncreds/src/ffi/cred_def.rs

```diff
@@ -1,22 +1,20 @@
-use std::os::raw::c_char;
+use std::ffi::c_char;
 
 use ffi_support::{rust_string_to_c, FfiStr};
-use indy_utils::Qualifiable;
 
 use super::error::{catch_error, ErrorCode};
-use super::object::{AnonCredsObjectId, ObjectHandle};
+use super::object::ObjectHandle;
 use crate::services::{
     issuer::create_credential_definition,
     types::{
-        CredentialDefinition, CredentialDefinitionConfig, CredentialDefinitionId,
-        CredentialDefinitionPrivate, CredentialKeyCorrectnessProof as KeyCorrectnessProof,
-        DidValue, SignatureType,
+        CredentialDefinition, CredentialDefinitionConfig, CredentialDefinitionPrivate,
+        CredentialKeyCorrectnessProof as KeyCorrectnessProof, SignatureType,
     },
 };
 
 #[no_mangle]
 pub extern "C" fn anoncreds_create_credential_definition(
-    origin_did: FfiStr,
+    schema_id: FfiStr,
     schema: ObjectHandle,
```

Comment:
Within the library we have an ObjectHandle which points to a mapping between handles and anoncredsobjects.

If we add a schema id to that it would be a weird mapping. Like, this would be a allowed, but quite vague.

```rust
struct AnonCredsobjectWithId {
  id: String,
  object: AnonCredsObject
}
```
But, my preference goes to the seperate schema_id, as it makes the ffi wrapper a lot easier to work with. Nested objects, structs, classes, etc. can get very annoying when sending them over an FFI layer and keeping it simple like this we avoid a lot of IMO unnecessary work.